# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/videoconduit_unittests.cpp
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 media/webrtc/signaling/gtest/videoconduit_unittests.cpp
--- a/media/webrtc/signaling/gtest/videoconduit_unittests.cpp	Fri Oct 23 13:49:51 2020 -0400
+++ b/media/webrtc/signaling/gtest/videoconduit_unittests.cpp	Fri Oct 23 14:02:54 2020 -0400
@@ -14,8 +14,8 @@
 #include "RtpRtcpConfig.h"
 #include "WebrtcGmpVideoCodec.h"
 
-#include "webrtc/media/base/videoadapter.h"
-#include "webrtc/media/base/videosinkinterface.h"
+#include "api/video/video_sink_interface.h"
+#include "media/base/video_adapter.h"
 
 #include "MockCall.h"
 
